_topic_map.yml

```diff
@@ -2037,6 +2037,8 @@ Distros: openshift-enterprise,openshift-origin
 Topics:
 - Name: About MTC
   File: about-mtc
+- Name: Release notes
```

Comment:
Please include the product name, project name, or some other way to identify these release notes as separate from the platform release notes.  As an example, see how [CI/CD pipelines](https://docs.openshift.com/container-platform/4.7/cicd/pipelines/op-release-notes.html) did it.